developing-without-a-build/article.md

```diff
@@ -0,0 +1,257 @@
+# Developing without a build
+In the early days of web development all we needed was a simple file editor and a web server. The development experience was really simple, and easy for a newcomer to understand and get started. Web development has changed a lot since then, the complexity of the tools we use for development has grown just as much as the complexity of the things that we're building on the web.
```

Comment:
```suggestion
In the early days of web development, all we needed was a simple file editor and a web server. It was easy for a newcomer to understand the process and get started making their own web pages. Web development has changed a lot since then: the complexity of the tools we use for development has grown just as much as the complexity of the things that we're building on the web.
```

Reply:
We actually need to configure source maps for development too :) Updating this one slightly